packages/inspire/Revelation.js

```diff
@@ -104,9 +105,17 @@ function _trySpread (candidate: any) {
   const options = candidate["..."];
   const rest = { ...candidate };
   delete rest["..."];
+  if (isInBrowser) {
+    return [
+      _markLazy(() => request(typeof options === "string" ? { url: options } : options)),
+      rest,
+    ];
+  }
+  const path = require("path");
+
   return [
-    _markLazy(() => request(typeof options === "string" ? { url: options } : options)),
-    rest,
+    require(path.join(process.cwd(), global.revelationPath, options)),
```

Comment:
As you asked: the _markLazy is a tool which allows delaying the actual request to only when the revelation consumer actually tries to access the particular property, instead of starting all requests immediately when creating the revelation. 
This is used by inspire revelations so that if the revelation refers to internet resources but those resources are already in the local cache, the internet requests are not made in the first place.
See InspireGateway.js:_connectAndNarratePrologue on where the consumption-or-skip is actually done.

For perspire stuff this is fine, loading local resources is ~free anyway.